deploy to staging

BacLuc opened the pull request.

manuelmeister committed:
Add custom color & abbr on CampCollaboration

renovate[bot] committed:
Merge pull request #5308 from ecamp/renovate/eslint-plugin-n-17.x

chore(deps): update dependency eslint-plugin-n to v17.8.1

renovate[bot] committed:
chore(deps): update dependency @sentry/vite-plugin to v2.18.0

renovate[bot] committed:
Merge pull request #5310 from ecamp/renovate/pulumi-pulumi-nodejs-3.x

chore(deps): update pulumi/pulumi-nodejs docker tag to v3.119.0

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.16.4

renovate[bot] committed:
Merge pull request #5316 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.16.4

renovate[bot] committed:
Merge pull request #5305 from ecamp/renovate/prettier-3.x

chore(deps): update dependency prettier to v3.3.1

renovate[bot] committed:
Merge pull request #5296 from ecamp/renovate/sentry-vite-plugin-2.x

chore(deps): update dependency @sentry/vite-plugin to v2.18.0

pmattmann committed:
Merge pull request #5288 from manuelmeister/fix/YSLogo

Harmonize Y+S logo on all channels & add icons in settings

pmattmann committed:
Merge pull request #5280 from usu/chore/print-logging

chore(print): improved request logging for nuxt-print

pmattmann committed:
fix PurgeHttpCacheListener
Ignore entities not exposed by the API

pmattmann reviewed (changes requested):
Wait for #5317
OAuth is not working

pmattmann committed:
fix unittest

BacLuc committed:
ecamp3-logging: switch to self built fluentd image

This is the image:
https://github.com/fluent/fluent-operator/blob/2de2b934e4d9f4475e278da7d9db74d89ef9a037/cmd/fluent-watcher/fluentd/Dockerfile.amd64
built in a devcontainer of the repository and then pushed to dockerhub.
The command to build the container was:
@BacLuc ➜ /workspaces/fluent-operator (master) $ docker build . -f cmd/fluent-watcher/fluentd/Dockerfile.amd64 -t docker.io/bacluc/fluentd:1.15.3

They seem to have problem with the images they publish in the fluent-operator repository.
It also contains the following fix: https://github.com/fluent/fluent-operator/pull/1195

BacLuc committed:
ops-dashboard: fix chart name

This was a copy and paste error.

renovate[bot] committed:
chore(deps): update tiryoh/gha-jobid-action action to v1.3.0

renovate[bot] committed:
fix(deps): update dependency uuid to v10

BacLuc committed:
Merge pull request #5321 from BacLuc/fix-ops-dashboard-name

ops-dashboard: fix chart name

renovate[bot] committed:
Merge pull request #5325 from ecamp/renovate/tiryoh-gha-jobid-action-1.x

chore(deps): update tiryoh/gha-jobid-action action to v1.3.0

renovate[bot] committed:
chore(deps): update helm release oauth2-proxy to v7.7.2

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.12.0

renovate[bot] committed:
Merge pull request #5327 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.12.0

renovate[bot] committed:
Merge pull request #5326 from ecamp/renovate/oauth2-proxy-7.x

chore(deps): update helm release oauth2-proxy to v7.7.2

renovate[bot] committed:
chore(deps): update dependency @typescript-eslint/eslint-plugin to v7.13.0

pmattmann committed:
add unittest

pmattmann committed:
Merge pull request #5317 from pmattmann/bugfix/PurgeHttpCacheListenser

fix PurgeHttpCacheListener

renovate[bot] committed:
Merge pull request #5328 from ecamp/renovate/typescript-eslint-monorepo

chore(deps): update dependency @typescript-eslint/eslint-plugin to v7.13.0

renovate[bot] committed:
chore(deps): update dependency @testing-library/jest-dom to v6.4.6

renovate[bot] committed:
chore(deps): lock file maintenance

renovate[bot] committed:
Merge pull request #5330 from ecamp/renovate/testing-library-monorepo

chore(deps): update dependency @testing-library/jest-dom to v6.4.6

renovate[bot] committed:
chore(deps): update dependency prettier to v3.3.2

renovate[bot] committed:
Merge pull request #5332 from ecamp/renovate/prettier-3.x

chore(deps): update dependency prettier to v3.3.2

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.12.1

renovate[bot] committed:
chore(deps): update dependency nuxt to v3.12.1

renovate[bot] committed:
Merge pull request #5333 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.12.1

renovate[bot] committed:
Merge pull request #5334 from ecamp/renovate/nuxtjs-monorepo

chore(deps): update dependency nuxt to v3.12.1

renovate[bot] committed:
fix(deps): update sentry-javascript monorepo to v8.9.1

renovate[bot] committed:
chore(deps): update dependency lint-staged to v15.2.6

renovate[bot] committed:
Merge pull request #5335 from ecamp/renovate/sentry-javascript-monorepo

fix(deps): update sentry-javascript monorepo to v8.9.1

renovate[bot] committed:
chore(deps): update dependency nuxt to v3.12.1

renovate[bot] committed:
chore(deps): update docker/dockerfile docker tag to v1.8

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.16.6

renovate[bot] committed:
Merge pull request #5336 from ecamp/renovate/lint-staged-15.x

chore(deps): update dependency lint-staged to v15.2.6

renovate[bot] committed:
fix(deps): update dependency @pulumi/aws to v6.39.1

renovate[bot] committed:
Merge pull request #5339 from ecamp/renovate/docker-dockerfile-1.x

chore(deps): update docker/dockerfile docker tag to v1.8

renovate[bot] committed:
Merge pull request #5340 from ecamp/renovate/pulumi-aws-6.x

fix(deps): update dependency @pulumi/aws to v6.39.1

renovate[bot] committed:
Merge pull request #5329 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.16.6

renovate[bot] committed:
chore(deps): update dependency node to v22.3.0

renovate[bot] committed:
Merge pull request #5338 from ecamp/renovate/node-22.x

chore(deps): update dependency node to v22.3.0

renovate[bot] committed:
fix(deps): update sentry-javascript monorepo to v8.9.2

renovate[bot] committed:
fix(deps): update dependency @pulumi/awsx to v2.12.0

renovate[bot] committed:
Merge pull request #5341 from ecamp/renovate/sentry-javascript-monorepo

fix(deps): update sentry-javascript monorepo to v8.9.2

renovate[bot] committed:
Merge pull request #5342 from ecamp/renovate/pulumi-awsx-2.x

fix(deps): update dependency @pulumi/awsx to v2.12.0

renovate[bot] committed:
fix(deps): update dependency @pulumi/awsx to v2.12.1

renovate[bot] committed:
chore(deps): update dependency lint-staged to v15.2.7

renovate[bot] committed:
Merge pull request #5343 from ecamp/renovate/pulumi-awsx-2.x

fix(deps): update dependency @pulumi/awsx to v2.12.1

renovate[bot] committed:
fix(deps): update dependency @pulumi/aws to v6.40.0

renovate[bot] committed:
Merge pull request #5344 from ecamp/renovate/lint-staged-15.x

chore(deps): update dependency lint-staged to v15.2.7

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.16.7

renovate[bot] committed:
Merge pull request #5345 from ecamp/renovate/pulumi-aws-6.x

fix(deps): update dependency @pulumi/aws to v6.40.0

renovate[bot] committed:
Merge pull request #5346 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.16.7

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.12.2

renovate[bot] committed:
Merge pull request #5348 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.12.2

renovate[bot] committed:
chore(deps): update helm release fluent-operator to v2.8.0

renovate[bot] committed:
fix(deps): update dependency @pulumi/pulumi to v3.120.0

renovate[bot] committed:
Merge pull request #5349 from ecamp/renovate/fluent-operator-2.x

chore(deps): update helm release fluent-operator to v2.8.0

renovate[bot] committed:
chore(deps): update actions/checkout digest to 692973e

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.16.8

renovate[bot] committed:
Merge pull request #5354 from ecamp/renovate/pulumi-pulumi-3.x

fix(deps): update dependency @pulumi/pulumi to v3.120.0

renovate[bot] committed:
Merge pull request #5331 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
Merge pull request #5356 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.16.8

renovate[bot] committed:
fix(deps): update dependency @pulumi/awsx to v2.12.1

renovate[bot] committed:
chore(deps): update dependency eslint-plugin-n to v17.9.0

renovate[bot] committed:
fix(deps): update dependency api-platform/core to v3.3.6

renovate[bot] committed:
Merge pull request #5357 from ecamp/renovate/eslint-plugin-n-17.x

chore(deps): update dependency eslint-plugin-n to v17.9.0

renovate[bot] committed:
Merge pull request #5358 from ecamp/renovate/api-platform-core-3.x

fix(deps): update dependency api-platform/core to v3.3.6

renovate[bot] committed:
chore(deps): update pulumi/pulumi-nodejs docker tag to v3.120.0

renovate[bot] committed:
Merge pull request #5350 from ecamp/renovate/pulumi-pulumi-nodejs-3.x

chore(deps): update pulumi/pulumi-nodejs docker tag to v3.120.0

renovate[bot] committed:
chore(deps): update dependency vite to v5.3.1

renovate[bot] committed:
chore(deps): update cypress-io/github-action action to v6.7.1

renovate[bot] committed:
chore(deps): update vue-minor-print-pdf to v3.4.29

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.16.9

renovate[bot] committed:
Merge pull request #5362 from ecamp/renovate/cypress-io-github-action-6.x

chore(deps): update cypress-io/github-action action to v6.7.1

renovate[bot] committed:
Merge pull request #5363 from ecamp/renovate/vue-minor-print-pdf

chore(deps): update vue-minor-print-pdf to v3.4.29

renovate[bot] committed:
Merge pull request #5364 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.16.9

renovate[bot] committed:
chore(deps): update dependency phpunit/phpunit to v10.5.21

BacLuc committed:
ecamp3-logging: split deploy command onto multiple lines

That it is more readable

BacLuc committed:
ecamp3-logging: allow to set different values for prod

renovate[bot] committed:
chore(deps): update helm release fluent-operator to v2.9.0

renovate[bot] committed:
chore(deps): update helm release oauth2-proxy to v7.7.3

renovate[bot] committed:
Merge pull request #5367 from ecamp/renovate/phpunit-phpunit-10.x

chore(deps): update dependency phpunit/phpunit to v10.5.21

renovate[bot] committed:
Merge pull request #5361 from ecamp/renovate/fluent-operator-2.x

chore(deps): update helm release fluent-operator to v2.9.0

renovate[bot] committed:
Merge pull request #5360 from ecamp/renovate/oauth2-proxy-7.x

chore(deps): update helm release oauth2-proxy to v7.7.3

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.12.2

BacLuc committed:
controls: use path instead of fieldname for select row

fieldname does not exist anymore.

manuelmeister committed:
Merge pull request #5174 from manuelmeister/fix/null-colorpicker

Create new color picker & color field

renovate[bot] committed:
Merge pull request #5355 from ecamp/renovate/vite-5.x

chore(deps): update dependency vite to v5.3.1

BacLuc committed:
Merge pull request #5369 from BacLuc/controls-fix-path

controls: use path instead of fieldname for select row

BacLuc committed:
Merge pull request #5352 from ecamp/renovate/actions-checkout-digest

chore(deps): update actions/checkout digest to 692973e

BacLuc committed:
Merge pull request #5337 from ecamp/renovate/nuxtjs-monorepo

chore(deps): update dependency nuxt to v3.12.2

BacLuc committed:
Merge pull request #5319 from BacLuc/fix-ecamp3-logging

ecamp3-logging: switch to self built fluentd image

BacLuc committed:
EColorPicker.spec.js: delete "accepts 3-digit hex color codes, after picker has been shown"

Because it is flaky now.

manuelmeister committed:
Merge pull request #5370 from BacLuc/skip-frontend-test

EColorPicker.spec.js: delete "accepts 3-digit hex color codes, after picker has been shown"

carlobeltrame committed:
Merge pull request #5366 from BacLuc/ecamp3-logging-increase-memory-for-prod

ecamp3-logging: increase memory for prod

renovate[bot] committed:
chore(deps): update dependency nuxt to v3.12.2

usu committed:
Merge pull request #5347 from ecamp/renovate/pulumi-awsx-2.x

fix(deps): update dependency @pulumi/awsx to v2.12.1

usu committed:
Merge pull request #5324 from ecamp/renovate/uuid-10.x

fix(deps): update dependency uuid to v10

usu committed:
Merge remote-tracking branch 'ecamp/devel' into chore/disable-symfony-listeners

renovate[bot] committed:
fix(deps): update symfony packages

usu committed:
merge devel

usu committed:
change to NameConverterInterface

BacLuc committed:
ecamp3-logging: switch back to fluent-operator image

They use ghcr.io/fluent/fluent-operator/fluentd:v1.17.0 with v2.9.0,
which seems to work again.

closes #5320

BacLuc committed:
ecamp3-logging: add instruction to diff the deploymentto README.md

usu committed:
fix phpstan

usu committed:
Merge pull request #5353 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.12.2

renovate[bot] committed:
chore(deps): update helm release oauth2-proxy to v7.7.4

renovate[bot] committed:
Merge pull request #5376 from ecamp/renovate/oauth2-proxy-7.x

chore(deps): update helm release oauth2-proxy to v7.7.4

BacLuc committed:
ops-dashboard: add instruction to diff the deploymentto README.md

BacLuc committed:
Merge pull request #5371 from ecamp/renovate/nuxtjs-monorepo

chore(deps): update dependency nuxt to v3.12.2

BacLuc committed:
CI: only run api checks if needed

And compare against result, not against status.
status is not here anymore.

BacLuc committed:
CI: only run e2e checks if needed

usu committed:
enable renovate updates for frankenphp

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.12.3

renovate[bot] committed:
chore(deps): update docker/build-push-action action to v6

carlobeltrame committed:
Handle absence of OAuth state cookie

carlobeltrame committed:
Return JSON on error

carlobeltrame committed:
Prolong available time for logging in to the third party system to 15mins

This should make
https://ecamp.sentry.io/issues/3531734868/?project=5912620 way rarer.

renovate[bot] committed:
Merge pull request #5383 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.12.3

renovate[bot] committed:
chore(deps): update dependency phpstan/phpstan to v1.11.5

pmattmann committed:
remove double base-constructor call

pmattmann committed:
remove unused trait

renovate[bot] committed:
chore(deps): update dependency @typescript-eslint/eslint-plugin to v7.13.1

renovate[bot] committed:
Merge pull request #5385 from ecamp/renovate/phpstan-packages

chore(deps): update dependency phpstan/phpstan to v1.11.5

renovate[bot] committed:
chore(deps): update dependency @types/node to v20.14.3

renovate[bot] committed:
Merge pull request #5386 from ecamp/renovate/typescript-eslint-monorepo

chore(deps): update dependency @typescript-eslint/eslint-plugin to v7.13.1

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.16.10

renovate[bot] committed:
Merge pull request #5387 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.14.3

renovate[bot] committed:
Merge pull request #5388 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.16.10

renovate[bot] committed:
chore(deps): update dependency @types/node to v20.14.4

renovate[bot] committed:
Merge pull request #5389 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.14.4

renovate[bot] committed:
chore(deps): update dependency @types/node to v20.14.5

usu committed:
Merge pull request #5285 from manuelmeister/feature/cursor-indicator-link

TipTap Autolink: Show cursor on hover + meta/ctrl

renovate[bot] committed:
Merge pull request #5390 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.14.5

usu committed:
Merge pull request #5384 from pmattmann/bugfix/minors

Minor php bugfixes

usu committed:
merge devel

usu committed:
Merge pull request #5373 from ecamp/renovate/symfony

fix(deps): update symfony packages

usu committed:
use extension of CacheManager as test mock

usu committed:
fix PurgeHttpCacheListenerTest

usu committed:
fix test namespaces + typos

BacLuc committed:
Merge pull request #5374 from BacLuc/eacmp3-logging-switch-back-to-stock-image

ecamp3-logging: switch back to fluent-operator image

renovate[bot] committed:
chore(deps): lock file maintenance

usu committed:
fix: change log output to single line

carlobeltrame committed:
Merge pull request #5382 from carlobeltrame/fix-expired-oauth-state-cookie

Fix expired oauth state cookie

renovate[bot] committed:
Merge pull request #5395 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
fix(deps): update dependency @pulumi/aws to v6.41.0

usu committed:
Merge pull request #5391 from usu/fix/test-namespaces

fix: test namespaces + some typos

usu committed:
Merge pull request #5381 from ecamp/renovate/docker-build-push-action-6.x

chore(deps): update docker/build-push-action action to v6

renovate[bot] committed:
Merge pull request #5396 from ecamp/renovate/pulumi-aws-6.x

fix(deps): update dependency @pulumi/aws to v6.41.0

usu committed:
Merge pull request #5276 from usu/chore/disable-symfony-listeners

chore(api): disable use_symfony_listeners

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.16.11

renovate[bot] committed:
chore(deps): update dependency cypress to v13.12.0

renovate[bot] committed:
Merge pull request #5397 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.16.11

renovate[bot] committed:
Merge pull request #5398 from ecamp/renovate/cypress-13.x

chore(deps): update dependency cypress to v13.12.0

renovate[bot] committed:
chore(deps): update dependency phpunit/phpunit to v10.5.22

renovate[bot] committed:
chore(deps): update cypress/included docker tag to v13.12.0

manuelmeister committed:
Rename abbr to abbreviation

manuelmeister committed:
Limit abbreviation to 1 emoji or 2 letters/numbers

manuelmeister committed:
Remove untested dead code

manuelmeister committed:
Hide abbreviation and color on create dialog as requested

renovate[bot] committed:
Merge pull request #5399 from ecamp/renovate/phpunit-phpunit-10.x

chore(deps): update dependency phpunit/phpunit to v10.5.22

renovate[bot] committed:
Merge pull request #5400 from ecamp/renovate/cypress-included-13.x

chore(deps): update cypress/included docker tag to v13.12.0

usu committed:
Merge pull request #5394 from usu/fix/print-single-line-log

fix: change log output to single line

manuelmeister committed:
Validate new campCollaboration color & abbreviation

manuelmeister committed:
Test if it takes the abbreviation and not the display name

manuelmeister committed:
Merge pull request #5281 from manuelmeister/feature/add-custom-color-campcollaboration

Add custom color campcollaboration

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.12.4

renovate[bot] committed:
chore(deps): update dependency @sentry/vite-plugin to v2.19.0

renovate[bot] committed:
fix(deps): update sentry-javascript monorepo to v8.10.0

renovate[bot] committed:
Merge pull request #5401 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.12.4

renovate[bot] committed:
Merge pull request #5402 from ecamp/renovate/sentry-vite-plugin-2.x

chore(deps): update dependency @sentry/vite-plugin to v2.19.0

renovate[bot] committed:
chore(deps): update dependency @types/node to v20.14.6

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.16.12

renovate[bot] committed:
Merge pull request #5404 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.14.6

renovate[bot] committed:
chore(deps): update dependency vimeo/psalm to v5.25.0

renovate[bot] committed:
Merge pull request #5403 from ecamp/renovate/sentry-javascript-monorepo

fix(deps): update sentry-javascript monorepo to v8.10.0

renovate[bot] committed:
Merge pull request #5405 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.16.12

renovate[bot] committed:
Merge pull request #5406 from ecamp/renovate/vimeo-psalm-5.x

chore(deps): update dependency vimeo/psalm to v5.25.0

renovate[bot] committed:
chore(deps): update dependency phpunit/phpunit to v10.5.24

renovate[bot] committed:
Merge pull request #5409 from ecamp/renovate/phpunit-phpunit-10.x

chore(deps): update dependency phpunit/phpunit to v10.5.24

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.17.0

renovate[bot] committed:
Merge pull request #5410 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.17.0

renovate[bot] committed:
chore(deps): update dependency rector/rector to v1.1.1

renovate[bot] committed:
Merge pull request #5412 from ecamp/renovate/rector-rector-1.x

chore(deps): update dependency rector/rector to v1.1.1

renovate[bot] committed:
fix(deps): update dependency @pulumi/awsx to v2.12.3

renovate[bot] committed:
chore(deps): update pulumi/pulumi-nodejs docker tag to v3.121.0

renovate[bot] committed:
Merge pull request #5416 from ecamp/renovate/pulumi-pulumi-nodejs-3.x

chore(deps): update pulumi/pulumi-nodejs docker tag to v3.121.0

renovate[bot] committed:
Merge pull request #5415 from ecamp/renovate/pulumi-awsx-2.x

fix(deps): update dependency @pulumi/awsx to v2.12.3

renovate[bot] committed:
fix(deps): update dependency @pulumi/pulumi to v3.121.0

renovate[bot] committed:
Merge pull request #5417 from ecamp/renovate/pulumi-pulumi-3.x

fix(deps): update dependency @pulumi/pulumi to v3.121.0

renovate[bot] committed:
chore(deps): update vue-minor-print-pdf to v3.4.30

pmattmann committed:
Merge pull request #5249 from usu/fix/purge-tag-tests

fix: add tests for observing purged tags + fix edge cases

renovate[bot] committed:
Merge pull request #5418 from ecamp/renovate/vue-minor-print-pdf

chore(deps): update vue-minor-print-pdf to v3.4.30

pmattmann committed:
Merge pull request #5377 from BacLuc/ops-dashboard-add-diff-instructions

ops-dashboard: add instruction to diff the deploymentto README.md

pmattmann committed:
Merge pull request #5380 from usu/fix/frankenphp-renovate

enables renovate updates for frankenphp

renovate[bot] committed:
chore(deps): update dependency @vue/test-utils to v2.4.6

renovate[bot] committed:
chore(deps): update dependency eslint-plugin-vue-scoped-css to v2.8.1

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.12.5

renovate[bot] committed:
Merge pull request #5421 from ecamp/renovate/eslint-plugin-vue-scoped-css-2.x

chore(deps): update dependency eslint-plugin-vue-scoped-css to v2.8.1

renovate[bot] committed:
Merge pull request #5422 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.12.5

renovate[bot] committed:
fix(deps): update sentry-javascript monorepo to v8.11.0

renovate[bot] committed:
Merge pull request #5413 from ecamp/renovate/sentry-javascript-monorepo

fix(deps): update sentry-javascript monorepo to v8.11.0

BacLuc committed:
Merge pull request #5379 from BacLuc/ci-skip-not-needed-tests

CI: skip not needed tests

renovate[bot] committed:
chore(deps): update dependency @sentry/vite-plugin to v2.20.0

renovate[bot] committed:
Merge pull request #5424 from ecamp/renovate/sentry-vite-plugin-2.x

chore(deps): update dependency @sentry/vite-plugin to v2.20.0

renovate[bot] committed:
chore(deps): update dependency @typescript-eslint/eslint-plugin to v7.14.1

renovate[bot] committed:
fix(deps): update dependency colorjs.io to v0.5.1

renovate[bot] committed:
Merge pull request #5425 from ecamp/renovate/typescript-eslint-monorepo

chore(deps): update dependency @typescript-eslint/eslint-plugin to v7.14.1

renovate[bot] committed:
fix(deps): update dependency nelmio/cors-bundle to v2.5.0

renovate[bot] committed:
Merge pull request #5426 from ecamp/renovate/colorjs.io-0.x

fix(deps): update dependency colorjs.io to v0.5.1

renovate[bot] committed:
Merge pull request #5427 from ecamp/renovate/nelmio-cors-bundle-2.x

fix(deps): update dependency nelmio/cors-bundle to v2.5.0

renovate[bot] committed:
chore(deps): update dependency friendsofphp/php-cs-fixer to v3.59.3

renovate[bot] committed:
Merge pull request #5368 from ecamp/renovate/friendsofphp-php-cs-fixer-3.x

chore(deps): update dependency friendsofphp/php-cs-fixer to v3.59.3

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.17.1

renovate[bot] committed:
Merge pull request #5429 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.17.1

renovate[bot] committed:
chore(deps): update helm release oauth2-proxy to v7.7.5

renovate[bot] committed:
Merge pull request #5430 from ecamp/renovate/oauth2-proxy-7.x

chore(deps): update helm release oauth2-proxy to v7.7.5

renovate[bot] committed:
chore(deps): update dependency unplugin-vue-components to v0.27.1

renovate[bot] committed:
chore(deps): update dependency @sentry/vite-plugin to v2.20.1

renovate[bot] committed:
Merge pull request #5432 from ecamp/renovate/unplugin-vue-components-0.x

chore(deps): update dependency unplugin-vue-components to v0.27.1

renovate[bot] committed:
chore(deps): update helm release oauth2-proxy to v7.7.6

renovate[bot] committed:
fix(deps): update sentry-javascript monorepo to v8.12.0

renovate[bot] committed:
Merge pull request #5434 from ecamp/renovate/sentry-vite-plugin-2.x

chore(deps): update dependency @sentry/vite-plugin to v2.20.1

renovate[bot] committed:
Merge pull request #5435 from ecamp/renovate/oauth2-proxy-7.x

chore(deps): update helm release oauth2-proxy to v7.7.6

BacLuc committed:
Merge pull request #5375 from BacLuc/diff-deployment

ecamp3-logging: add instruction to diff the deploymentto README.md

renovate[bot] committed:
Merge pull request #5437 from ecamp/renovate/sentry-javascript-monorepo

fix(deps): update sentry-javascript monorepo to v8.12.0

BacLuc committed:
api: apply cs-fix

https://github.com/ecamp/ecamp3/pull/5368 was merged accidentally.
Maybe because of https://github.com/ecamp/ecamp3/issues/4309

BacLuc commented:
Needs
- https://github.com/ecamp/ecamp3/pull/5438

BacLuc committed:
revert "chore(deps): update dependency @vue/test-utils to v2.4.6" in frontend

renovate[bot] committed:
fix(deps): update dependency colorjs.io to v0.5.2

renovate[bot] committed:
fix(deps): update dependency @pulumi/aws to v6.42.0

carlobeltrame committed:
Merge pull request #5142 from ecamp/renovate/vue-test-utils-2.x

chore(deps): update dependency @vue/test-utils to v2.4.6

renovate[bot] committed:
Merge pull request #5441 from ecamp/renovate/pulumi-aws-6.x

fix(deps): update dependency @pulumi/aws to v6.42.0

renovate[bot] committed:
Merge pull request #5440 from ecamp/renovate/colorjs.io-0.x

fix(deps): update dependency colorjs.io to v0.5.2

renovate[bot] committed:
chore(deps): update dependency vite to v5.3.2

renovate[bot] committed:
chore(deps): update helm release oauth2-proxy to v7.7.8

renovate[bot] committed:
Merge pull request #5442 from ecamp/renovate/vite-5.x

chore(deps): update dependency vite to v5.3.2

renovate[bot] committed:
Merge pull request #5443 from ecamp/renovate/oauth2-proxy-7.x

chore(deps): update helm release oauth2-proxy to v7.7.8

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.17.4

renovate[bot] committed:
chore(deps): update vue-minor-print-pdf to v3.4.31

renovate[bot] committed:
Merge pull request #5447 from ecamp/renovate/vue-minor-print-pdf

chore(deps): update vue-minor-print-pdf to v3.4.31

pmattmann committed:
Merge pull request #5438 from BacLuc/apply-cs-fix

api: apply cs-fix

renovate[bot] committed:
Merge pull request #5431 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.17.4

BacLuc commented:
/fast-forward
